ui/src/common/layouts/Header/__tests__/BannerContainer.test.jsx

```diff
@@ -34,25 +31,7 @@ describe('BannerContainer', () => {
         </MemoryRouter>
       </Provider>
     );
-    expect(getByText('Test')).toBeInTheDocument();
-  });
 
-  it('dispatches UI_CLOSE_BANNER with banner id, on banner close', () => {
```

Comment:
this one here ?

Reply:
I was not able to rewrite this test case (the action should be released on animation end (afterClose prop), but I was not able to catch it with rtl. I changed the test case to see if the button closes banner properly